andela/activo-web
# #159569285 Create default loading message while asset categories are being fetched on dashboard

**What does this PR do?**
Creates a graphic spinner while asset categories are being loaded on the dashboard

**Description of Task to be completed?**
- create spinner component
- write actions type to listen for requests to fetch asset categories
- write reducer case for request action dispatched
- write tests for spinner logic

**How should this be manually tested?**
After cloning, run the app by running `yarn start:dev`
- While the product is trying to make an API call, you should see a simple spinner loading.

**Any background context you want to provide?**
Users shouldn't experience a lag in the application due to the time it takes to fetch data, as such, a loader needed to be implemented.

**What are the relevant pivotal tracker stories?**
[#159569285](https://www.pivotaltracker.com/story/show/159569285)

**Screenshots (if appropriate)**
<img width="1440" alt="screen shot 2018-08-06 at 8 50 27 pm" src="https://user-images.githubusercontent.com/24798364/43757771-d961c884-9a11-11e8-8766-124ed941ac8f.png">


**Questions:**
N/A
